tests/functional_test.js

```diff
@@ -222,8 +214,7 @@ describe('functional', function () {
                 return;
             }
 
-            var domain = helpers.domainCheck('https://maxcdn.bootstrapcdn.com/');
-            var uri = domain + root + '/' + name;
+            var uri = 'https://maxcdn.bootstrapcdn.com/' + root + '/' + name;
```

Comment:
This wasn't hardcoded, because I wanted to be able to test S3 separately when run locally, to debug inconsistencies.